Add custom metric - observability

dockerymick opened the pull request:
https://github.com/open-cluster-management/backlog/issues/6428

dockerymick committed:
Add custom metric - observability

dockerymick committed:
Update manage_observe.adoc

updating the placement of the section

dockerymick committed:
Update manage_observe.adoc

dockerymick committed:
Update manage_observe.adoc

Updating the note details and placement for automatic pod restarts  when custom rule is updated 
https://github.com/open-cluster-management/backlog/issues/7202

clyang82 commented on observability/manage_observe.adoc:
```diff
@@ -54,6 +53,48 @@ data:
+[#adding-custom-metrics]
+=== Adding custom metrics
+
+Add metrics to the `metrics_list.yaml` file, in order to view collected metrics that are from managed clusters. *Note*: When you update your custom rules, `observability-observatorium-thanos-rule` pods are restarted automatically.
```
Why you mention update your custom rules? This is for customizing metrics. not for customizing the rules.

dockerymick replied:
@clyang82 Should Adding custom metrics have a lower Header? H3 to H2. In the file, I would remove an equal sign (=) on line 57

dockerymick committed:
Update manage_observe.adoc

Small update for consistency purposes. Verified that all links are functioning as aspected

dockerymick commented:
@dislbenn Thank you for the review!

clyang82 commented on observability/manage_observe.adoc:
```diff
@@ -54,6 +52,48 @@ data:
+----
+
+. Access the `observability-metrics-custom-whitelist.yaml` file in the 
+`open-cluster-management-observability` namespace by running the following command:
```
by default, there is no `observability-metrics-custom-whitelist` configmap in observability namespace. the user needs to create this configmap, for example:
```
kind: ConfigMap
apiVersion: v1
metadata:
  name: observability-metrics-custom-whitelist
  namespace: open-cluster-management-observability
data:
  metrics_list.yaml: |
    names:
      - node_memory_Active_bytes
```

clyang82 commented:
@dockerymick sorry for late response. I just found that I forgot the click "submit" my reviews so that the comments are in Pending status.

dockerymick commented:
@clyang82 no worries, thank you for your comments, I will apply the suggestion in a different PR
